src/components/CheckoutLogin/ResetPasswordForm.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { PasswordResetForm } from "..";
+
+const ResetPasswordForm = ({ onClick }) => (
+  <>
+    <h3 className="checkout__header">Registered user</h3>
+    <PasswordResetForm />
+    <div className="login__content__password-reminder">
```

Comment:
This `BEM` looks really weird

Reply:
Done

Reply:
Added generic u-link class which is ussed in few places